dietpi/login

```diff
@@ -38,7 +38,7 @@
 	if (( $DIETPI_INSTALL_STAGE == 1 )); then
 
 		#Boot to specific Program
-		if [ -z "$DISPLAY" ] && [ $(tty) == /dev/tty1 ]; then
+		if [ -z "$DISPLAY" ] && [ "$(tty)" == /dev/tty1 ]; then
```

Comment:
I'll also remove the double parentheses as this is not arithmetic, and, set to double quotes on both input and output values.

if [ "$(tty)" = "/dev/tty1" ]; then